### ISSUE TITLE
Investigate ffmpeg delivery options

### ISSUE THREAD

**Issue by mcwhitaker** [avalonmediasystem/avalon#1251]
Xenial delivers ffmpeg packaged already whereas Avalon delivers ffmpeg as source code that needs to be built at the time of installation process. We thought the Avalon way was legally required.

We need to determine how we will handle ffmpeg with the packaging of r6.

**Comment by joncameron** [comment]
This will be relevant if we're using Docker for packaging in 6.0.

**Comment by davidschober** [comment]
NOTE: ubuntu licensing https://www.ubuntu.com/about/about-ubuntu/licensing. Note "main and restricted" which requires all packages to be redistributable.

Also, MBK pointed out http://www.webupd8.org/2014/11/ffmpeg-returns-to-official-ubuntu.html

So it's been in the distro for around 2 years.

**Comment by mcwhitaker** [comment]
I did some research on this based on the links above and also discussed with David. We concluded we are fine based on this:

1. Based on http://packages.ubuntu.com/xenial/libs/ and http://packages.ubuntu.com/xenial-updates/libs/, all ffmpeg libraries are part of the _universe_ repository.
2. Based on https://help.ubuntu.com/community/Repositories/Ubuntu, libraries distributed via the _universe_ ubuntu repository are _Community-maintained free and open-source software_.
3. Based on https://www.ubuntu.com/about/about-ubuntu/our-philosophy, ubuntu follow's the GNU definition of free software, of which freedom 2 is the freedom to redistribute copies.
